src/components/App.js

```diff
@@ -0,0 +1,13 @@
+import ButtonPanel from './ButtonPanel';
+import Display from './Display';
+
+function App() {
+  return (
+    <div className="App">
+      <Display />
+      <ButtonPanel />
+    </div>
+  );
+}
```

Comment:
- [ ] This component should have children components wrapped in `React.Fragment` short syntax.

[OPTIONAL]

- [ ] Kindly favor the use of ES6 syntax. In this case, please use the arrow function rather than the `function` keyword.